src/components/IssuesItem/IssuesItem.tsx

```diff
@@ -8,6 +8,8 @@ type IssuesItemProps = {
 };
 
 const IssuesItem: FC<IssuesItemProps> = ({ item, onIssuePress }) => {
+  console.log("IssuesItem");
```

Comment:
let's create a util for this `logRender` so we can easily tell apart random console.logs and the ones used to track renders.